extensions/amp-script/amp-script.md

```diff
@@ -188,21 +187,19 @@ Include the script hash in a `meta[name=amp-script-src]` element in the document
 2. base64url-encode the result.
 3. Prefix that with `sha384-`.
 
-Here's you might build the hash in node.js:
+Here's you might build the hash in Node.js using the [@ampproject/toolbox-script-csp](https://www.npmjs.com/package/@ampproject/toolbox-script-csp) package:
 
 ```js
-function generateCSPHash(script) {
```

Comment:
Good idea to suggest that people do this by importing the toolbox. Especially as then they'll be up-to-date if somehow we ever change the way this is calculated. I did also change the docs to mention the toolbox earlier.

That said, I think people should also know how to do this themselves - and providing the sample code is more explicit than the instructions I gave.

Why don't we include both methods?

Reply:
Developers should use the toolbox library instead of implementing it on their own. This gives us an easier upgrade path if we'd decide to change the implementation in the future. If they want to see the implementation they can look at the source code of the NPM module.